src/Humanizer/TimeSpanHumanizeExtensions.cs

```diff
@@ -162,15 +380,15 @@ private static string BuildFormatTimePart(IFormatter cultureFormatter, TimeUnit
                 ? cultureFormatter.TimeSpanHumanize(timeUnitType, Math.Abs(amountOfTimeUnits))
                 : null;
         }
-
-        private static List<string> CreateTimePartsWithNoTimeValue(string noTimeValue)
+        
+        private static bool IsContainingOnlyNullValue(IEnumerable<string> timeParts)
         {
-            return new List<string>() { noTimeValue };
+            return (timeParts.Count(x => x != null) == 0);
         }
 
-        private static bool IsContainingOnlyNullValue(IEnumerable<string> timeParts)
+        private static List<string> CreateTimePartsWithNoTimeValue(string noTimeValue)
         {
-            return (timeParts.Count(x => x != null) == 0);
+            return new List<string>() { noTimeValue };
```

Comment:
Okay, it's been reverted.
The reason why I changed that was because IsContainingOnlyNullValue was getting called first in the flow of the code.